src/borg/testsuite/archiver.py

```diff
@@ -79,6 +79,7 @@ def exec_cmd(*args, archiver=None, fork=False, exe=None, **kw):
                 archiver = Archiver()
             archiver.prerun_checks = lambda *args: None
             archiver.exit_code = EXIT_SUCCESS
+            helpers.return_code = EXIT_SUCCESS
```

Comment:
oh, i didn't notice archiver is an Archiver instance here.

what I meant was archiver module.